.travis.yml

```diff
@@ -40,13 +40,17 @@ env:
     - RUN_E2E_TESTS_STATE_EDITOR=true
     - RUN_E2E_TESTS_SUBSCRIPTIONS=true
     - RUN_E2E_TESTS_USERS=true
-    # E2E tests for mobile.
-    - RUN_E2E_TESTS_MOBILE=true
 
 matrix:
   allow_failures: []
   fast_finish: true
 
+jobs:
+  include:
+    # The mobile end-to-end tests are run only for the develop branch. 
```

Comment:
Perhaps explain why and link to documentation.

Reply:
Done, thanks!